searcher.js

```diff
@@ -0,0 +1,60 @@
+var ld = require('lodash');
+var storage = require('./storage.js');
+var padDB = require('./model/pad.js');
+
+module.exports = (function () {
+    'use strict';
+
+    var _bulkRetrieve = function(prefix, list, callback) {
+        var keys = list.map(id => prefix + id);
+        storage.fn.getKeys(keys, callback);
+    }
+
+    var searcher = {};
+
+    searcher.fn = {};
+
+    searcher.fn.searchPads = function (searchQuery, callback) {
+        console.log("searched for: " + searchQuery);
+        if (storage.db && (storage.db.type === 'postgres' || storage.db.type === 'postgrespool')) {
+            var query = 
+                "SELECT SUBSTRING(store.key, 12) AS key, store.value AS value, subquery.headline AS headline " +
```

Comment:
Correct to use parameterized query ($1, $2, $3).  But to create a multiline string in JavaScript, its easier if you use JavaScript template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals